instrumentation/delayed_job/test/test_helper.rb

```diff
@@ -24,31 +29,8 @@
   c.add_span_processor span_processor
 end
 
-ActiveRecord::Migration.verbose = false
-
-module TestHelper
-  extend self
-
-  def setup_active_record
-    ::ActiveRecord::Base.establish_connection(adapter: 'sqlite3', database: ':memory:')
-    ::ActiveRecord::Schema.define do
-      create_table 'delayed_jobs', force: :cascade do |t|
-        t.integer 'priority', default: 0, null: false
-        t.integer 'attempts', default: 0, null: false
-        t.text 'handler', null: false
-        t.text 'last_error'
-        t.datetime 'run_at'
-        t.datetime 'locked_at'
-        t.datetime 'failed_at'
-        t.string 'locked_by'
-        t.string 'queue'
-        t.datetime 'created_at'
-        t.datetime 'updated_at'
-      end
-    end
-  end
-
-  def teardown_active_record
-    ::ActiveRecord::Base.connection.close
-  end
-end
+gem_dir = Gem::Specification.find_by_name('delayed_job').gem_dir
+require "#{gem_dir}/spec/delayed/serialization/test"
+require "#{gem_dir}/spec/delayed/backend/test"
```

Comment:
IMHO it would make sense to add PR to delayed job to move this file into lib folder since it is useful for testing depending libs. Should I try to open PR?

Reply:
That would be great! Thank you

Reply:
:pray: https://github.com/collectiveidea/delayed_job/pull/1195

Reply:
Lead time seems like it is about 3-4 months IMO https://github.com/collectiveidea/delayed_job/pull/1169

Reply:
definitely not blocker from my side